components/composed/contributor/ContributorUpdatePersonForm/ContributorUpdatePersonForm.tsx

```diff
@@ -49,6 +50,7 @@ export default function ContributorUpdatePersonForm({ data }: Props) {
       mutation={mutation}
       getErrors={getErrors}
       toVariables={toVariables}
+      defaultValues={{ links }}
```

Comment:
This is a bit frustrating... in order to get the correct default value to useFieldArray, the default must be set at the form level. (You cannot set a default value at the field array level.)

Reply:
The way the default value is set on other forms might be unnecessary, and it's possible that we should be passing the default value to the parent form, _not_ the child inputs.